cheatSheet/README.md

```diff
@@ -21,9 +21,48 @@ Attention **Graphical Designers** I need you to make this cheat sheet look like
 Name <---\*--
 ```
 
+## Patterns
+
+* A free variable
+* `Bool` `Int` `String` `Uri` `ByteArray` Type patterns
+* `[ Head ... Tail ]` `Set( Head ... Tail )` where `Tail` is a variable
```

Comment:
Can you really do head and tail on Sets? They are unordered, right?

Reply:
Hmm, I'm testing this, see discord for the confusion. I'm thinking that `Head` and `Tail` probably aren't the best words to use there. My impression is that if you have a pattern like `Set( 1 , x )` you want to check that `1` is in the set and then bind `x` to the rest.